### DISCUSSION TITLE
Is Munt included?

### DISCUSSION THREAD

**Discussion by lavadrop** [86Box/86Box#2250] [Q&A] [answered]
Because if it's supposed to, it's missing from the AppImage library directories.
I can't get any sound from an emulated 486. LSL1 complains it can't initialize the audio hardware. LSL3 is just silent. Windows is the same, there's no sound.

**Comment by OBattler** [comment, marked as answer]
Munt is compiled directly into the binary (it's in the source tree). Did you download the required ROM's from our ROM's repository?

**Reply by lavadrop** [reply]
Yes, I downloaded all the roms from the repository, and I can confirm that both the MT32 and the CM32 roms are in the roms folder.

**Comment by jriwanek** [comment]
Question as posed answered, if you're still having issues, open a bug report.
